### PR TITLE
[Zeta Test] proposed Zeta test change

### PR DESCRIPTION
**Pull request by er1cthe0ne**
This change include:

1. updated the current DISABLED_auxgateway_test zeta test and confirmed passing
2. added test cases: DISABLED_zeta_gateway_path_CHILD and DISABLED_zeta_gateway_path_PARENT
3. added json parser and code to aca_tests
4. added test controller python code

### CODE REVIEW

**Comment by er1cthe0ne** [comment]
@zhangml - we fixed up the test case DISABLED_auxgateway_test and updated with a valid goal state (include Zeta gateway). We are seeing the below failure, please take a look. Note that this branch is based on @PikaPikaW 's change and we make fixes on top of it.

ACA_Vlan_Manager::get_aux_gateway_id <--- Exiting, auxGateway_id =
auxGateway_id is empty!
Executing command: ovs-ofctl -O OpenFlow13 dump-groups br-tun | grep group_id=1
Command failed!!! rc: 256
 Elapsed time for system command took: 5674 microseconds or 5 milliseconds.
ACA_OVS_L2_Programmer::execute_openflow_command ---> Entering
Executing command: ovs-ofctl -O OpenFlow13 add-group br-tun group_id=1,type=select,bucket=output:vxlan-40668915,bucket=output:vxlan-60637580
ovs-ofctl: vxlan-40668915: output to unknown port

File: test/gtest/aca_test_zeta_programming.cpp [hunk 1]
```diff
@@ -142,3 +172,21 @@ TEST(zeta_programming_test_cases, DISABLED_auxgateway_test)
...
+  // do some validate
+}
+
+TEST(zeta_programming_test_cases, DISABLED_zeta_gateway_path_PARENT)
+{
+  // TODO: The relative path of the PARENT configuration file
+  string zeta_gateway_path_PARENT_config_file = "./...";
+  aca_test_zeta_setup(zeta_gateway_path_PARENT_config_file);
+
+  // do some validate
```
Comment (zzxgzgz) [inline test/gtest/aca_test_zeta_programming.cpp:191]: @PikaPikaW please implement the testcases, thank you.

File: test/gtest/aca_test_zeta_programming.cpp [hunk 2]
```diff
@@ -103,42 +247,68 @@ TEST(zeta_programming_test_cases, delete_zeta_config_valid)
...
   EXPECT_EQ(retcode, EXIT_SUCCESS);
 }
+
+TEST(zeta_programming_test_cases, DISABLED_zeta_gateway_path_CHILD)
+{
+  // TODO: The relative path of the CHILD configuration file
+  string zeta_gateway_path_CHILD_config_file = "./test/gtest/aca_data.json";
+  aca_test_zeta_setup(zeta_gateway_path_CHILD_config_file);
+
+  // do some validate
```
Comment (er1cthe0ne) [inline test/gtest/aca_test_zeta_programming.cpp:304]: on child side, please check if the expected rules are installed.

File: test/gtest/aca_test_zeta_programming.cpp (hunk 1 above)
Comment (er1cthe0ne) [reply test/gtest/aca_test_zeta_programming.cpp:313]: on parent side, please check if the expected rules are installed, and then try to do a ping from parent port to child port, confirm the ping is successful.

**Comment by zzxgzgz** [comment]
@HuaqingTu @PikaPikaW 

I updated the pseudo controller, please pull my changes before further development. 

Thank you.

**Review by zzxgzgz — APPROVED** [review]
went through the change with community members, confirmed this is ready to merge

**Comment by er1cthe0ne** [comment]
closing this test only PR (for CI).

**Pull request closed without merging**